src/components/CommentForm.js

```diff
@@ -110,8 +134,15 @@ export class CommentForm extends Component {
               Enter text
             </p>
           )}
-          <input type="submit" className="commentButton" value="Add Comment" />
+
+          {!this.props.currentComment && <input type="submit" className="commentButton" value="Add Comment" />}
```

Comment:
`currentComment` should not be here. This form should always have these buttons.

Reply:
if i delete that condition whenever i want to edit a comment the 'Add Comment' button will be there too

Reply:
why don't you use your newly created method: `isEditModeOn()`?